boards/msb-430/include/board.h

```diff
@@ -43,8 +43,8 @@ extern "C" {
 /**
  * @brief   Define the CPU model for the <msp430.h>
  */
-#ifndef __MSP430F1612__
-#define __MSP430F1612__
+#ifndef MSP430F1612
```

Comment:
Yes, I got it. 

By the way, what about trailing underscores (e.g., ```MSB_BOARD_H_``` )? We just ignore trailing underscores without revising them??  (since they are not against C standard)